### PR TITLE
i18n(zh-cn): Translate recipes.mdx 

### PR DESCRIPTION
**Pull request by ocsxxi**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translate `recipes.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/recipes.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: 更多方案
+description: 为Astro添加特性的示例。
+
+---
+
+import RecipesNav from '~/components/RecipesNav.astro';
+
+为你的Astro项目中添加特性的示例。
```
Comment (luoingly) [inline src/content/docs/zh-cn/recipes.mdx:9]: This sentence seems to be grammatically incorrect. It should be modified to:
```
为你的 Astro 项目添加特性的示例。
```
or
```
在你的 Astro 项目中添加特性的示例。
```

File: src/content/docs/zh-cn/recipes.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## 官方方案
+
+<RecipesNav />
+
+## 社区资源
+
+晒出你的方案! 更多详情请查看 [recipes contributing guide](https://github.com/withastro/docs/blob/main/RECIPES.md)。
```
Comment (luoingly) [inline src/content/docs/zh-cn/recipes.mdx:18]: The exclamation mark should be represented by Chinese punctuation, and the link description should also be translated:
```suggestion
晒出你的方案！更多详情请查看[示例贡献指南](https://github.com/withastro/docs/blob/main/RECIPES.md)。
```

File: src/content/docs/zh-cn/recipes.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- [Prisma与Planetscale鉴权](https://www.thomasledoux.be/blog/authentication-astro-prisma-planetscale)
+- [通过动态文件名引入图片](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
+- [用Swup添加页面过渡动画](https://navillus.dev/blog/astro-plus-swup)
+- [在Astro中使用UnoCSS](https://www.elian.codes/blog/23-02-11-implementing-unocss-in-astro/)
+- [在Astro中通过Markdown标题构建目录](https://kld.dev/building-table-of-contents/)
+- [创建一个用于从Markdown文件中删除运行时的Remark插件](https://eatmon.co/blog/remove-runts-markdown/)
```
Comment (luoingly) [inline src/content/docs/zh-cn/recipes.mdx:25]: Inaccurate translation.

阅读了[这里提到的文章](https://eatmon.co/blog/remove-runts-markdown/)后就可以知道，"Runts" 在这里指的是段落中独自一行的最后一个词，我觉得可以翻译为 "孤立词"。（TL;DR: "Runts" should not be translated to "运行时".）

After reorganizing the expression:

```suggestion
- [创建用于删除 Markdown 文件中孤立词的 Remark 插件](https://eatmon.co/blog/remove-runts-markdown/)
```

**Review by luoingly — CHANGES_REQUESTED** [review]
Thank you for contributing to this project! 💜

But I think there are still changes that need to be made in your PR, so please check it out!

**Comment by ocsxxi** [comment]
@luoingly Thank you for pointing out these errors. I'd work harder on my language grammar. 🌹

感谢指出了这些翻译问题，这也从侧面体现了我的汉语语法还有待加强 😂

**Review by luoingly — APPROVED** [review]
LGTM. 💜

> @luoingly Thank you for pointing out these errors. I'd work harder on my language grammar. 🌹

They are just tiny problems, keep up the good work!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks @ocsxxi (welcome to Team Docs!) and @luoingly for the careful review 💜

**Pull request merged**